### ISSUE TITLE
Contact data should be nullable

### ISSUE THREAD

**Issue by alexander-schranz** [sulu-io/sulu#1322]
I think the most data in the contacts/users should be nullable

Currently the following country data is not nullable:

 - firstName
 - lastName 
 - address countryId
 - address type

**Comment by danrot** [comment]
@berndhep Do you agree? IMO first and last name is the least information being available, but I can imagine that on many imports this data is missing.

**Comment by berndhep** [comment]
First and last name should be required information in my opinion. Everything else can be null.

**Comment by berndhep** [comment]
Ok, I revoke what I have said. First/last name OR email address should be required.

**Comment by alexander-schranz** [comment]
we had the last times in the slack channel some requests how to make the firstname and lastname nullable for the community bundle. I would like to make this fields nullable in the core to be more developer friendly. /cc @sulu/core-team

**Comment by wachterjohannes** [comment]
I don't agree that - it is possible to overwrite the entity (maybe we have to document that) so everyone can change this easily. When we want this fields required in the UI of the admin we should also make them required in the entity (or vice versa if the opinion changed about required fields).
